README.md

```diff
@@ -1,3 +1,79 @@
 # Mollie PHP Coding Standards
 
 Contains PHP coding standards like rules for PHP-CS-Fixer that serves for purpose of standardization.
+
+## Usage
+
+Place a file named `.php_cs.dist` that has following content in your project's root directory.
+
+```php
+use Mollie\PhpCodingStandards\PhpCsFixer\Rules;
+use PhpCsFixer\Config;
+use PhpCsFixer\Finder;
+
+$finder = Finder::create()
+    ->name('.php_cs.dist')       // Fix this file as well
```

Comment:
Can you remove the superfluous spaces? They're here for alignment reasons when having multiple lines with multiple comments, but this example doesn't have all those, so all the extra spaces are unnecessary.